checks/raw/sast.go

```diff
@@ -159,100 +162,38 @@ func codeQLInCheckDefinitions(c *checker.CheckRequest) ([]checker.SASTWorkflow,
 				Offset: checker.OffsetDefault,
 				Type:   finding.FileTypeSource,
 			},
-			Type: checker.CodeQLWorkflow,
+			Type: checkerType,
 		}
 
 		sastWorkflows = append(sastWorkflows, sastWorkflow)
 	}
 	return sastWorkflows, nil
 }
 
-// Check file content.
-var searchGitHubActionWorkflowCodeQL fileparser.DoWhileTrueOnFileContent = func(path string,
+var searchGitHubActionWorkflowUseRegex fileparser.DoWhileTrueOnFileContent = func(path string,
 	content []byte,
 	args ...interface{},
 ) (bool, error) {
 	if !fileparser.IsWorkflowFile(path) {
 		return true, nil
 	}
 
-	if len(args) != 1 {
+	if len(args) != 2 {
 		return false, fmt.Errorf(
-			"searchGitHubActionWorkflowCodeQL requires exactly 1 arguments: %w", errInvalid)
+			"searchGitHubActionWorkflowUseRegex requires exactly 1 arguments: %w", errInvalid)
```

Comment:
error message says 1 arg, needs to be updated

Reply:
Fixed.